user-tool/tool/src/java/org/sakaiproject/user/tool/UsersAction.java

```diff
@@ -1118,17 +1121,18 @@ else if ((mode != null) && (mode.equalsIgnoreCase("edit")) && (!singleUser))
 
 		// get the user
 		UserEdit user = (UserEdit) state.getAttribute("user");
+		//if user has not changed the email then skip the 'email exists' verification
+		if(user != null && !(user.getEmail().equals(email))){
 			try {
 				UserDirectoryService.getUserByEid(email);
-				//If we have shown this 'email exists' message once and user still wants to create account with the same email address, continue with the process
-				String alertMessage = StringUtils.trimToNull(data.getParameters().getString("alert_message"));
-				if(!(alertMessage != null && alertMessage.equals(rb.getString("useedi.email.exists")))){
-					addAlert(state,rb.getString("useedi.email.exists"));
-					return false;
-				}
+				addAlert(state,rb.getString("useedi.email.exists"));
```

Comment:
This also prevents admins from changing a user's email address to one that is already used doesn't it?